lib/Models/UserDrawing.ts

```diff
@@ -192,9 +191,15 @@ export default class UserDrawing extends CreateModel(EmptyTraits) {
                 this.pointEntities.entities.values.length < 1
               )
                 return;
-              const point1 = this.pointEntities.entities.values[0].position.getValue(
-                time
-              ) as Cartesian3;
+              const point1 =
```

Comment:
Ah I think I was just trying to minimise the number of `?` used since Kevin didn't seem to use it that much in his PR, so I just wanted to follow the way he did the error fixing as much as possible. This one is inconsistent with `point2` 😆  I'll fix it for consistency

Reply:
Ignore everything I said above I clearly don't remember what I was talking about! So I don't think we can use `?` because `point1` can't be undefined for `Cartographic.fromCartesian(point1)` in the return statement. The function can only take in defined values and if we were to use `?` we _could_ potentially get undefined values and TS throws an error 😬